### PR TITLE
include official write ins in tally reports

### PR DESCRIPTION
**Pull request by carolinemodic**
## Overview
Note that I'm not seeing adjudicatedOptionId always set properly but assuming it is this will include those tallies in our regular tally reports and subtract them from the write-ins row. I don't love the way I did this, or the lack of testing, we should revisit this after the demo, and also make it a configurable thing since I imagine some jurisdictions won't want this. 

## Testing Plan 
Manually adjudicated as real candidates, saw the tally report adjust accordingly. 

## Checklist
- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [ ] I have added JSDoc comments to any newly introduced exports

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: frontends/election-manager/src/lib/votecounting.ts [hunk 1]
```diff
@@ -406,6 +409,57 @@ function processCastVoteRecord({
   return newCvr;
 }
 
+export function modifyTallyWithWriteInInfo(
+  tally: Tally,
+  writeIns: Map<string, Map<string, number>>
+): Tally {
+  console.log('start of call', writeIns);
```
Comment (benadida) [inline frontends/election-manager/src/lib/votecounting.ts:416]:
```suggestion
```

**Review by benadida — APPROVED** [review]

File: frontends/election-manager/src/components/election_manager_tally_report.tsx [hunk 2]
```diff
@@ -38,6 +39,7 @@ export interface Props {
   precinctId?: string;
   scannerId?: string;
   votingMethod?: VotingMethod;
+  officialCandidateWriteIns?: Map<string, Map<string, number>>; // Contest -> Candidate ID -> Count
```
Comment (eventualbuddha) [inline frontends/election-manager/src/components/election_manager_tally_report.tsx:42]:
```suggestion
  officialCandidateWriteIns?: Map<ContestId, Map<CandidateId, number>>;
```

File: frontends/election-manager/src/lib/votecounting.ts [hunk 3]
```diff
@@ -406,6 +409,56 @@ function processCastVoteRecord({
   return newCvr;
 }
 
+export function modifyTallyWithWriteInInfo(
+  tally: Tally,
+  writeIns: Map<string, Map<string, number>>
```
Comment (eventualbuddha) [inline frontends/election-manager/src/lib/votecounting.ts:414]: Ditto here. Maybe give it a type alias?

File: frontends/election-manager/src/lib/votecounting.ts [hunk 4]
```diff
@@ -406,6 +409,56 @@ function processCastVoteRecord({
   return newCvr;
 }
 
+export function modifyTallyWithWriteInInfo(
```
Comment (eventualbuddha) [inline frontends/election-manager/src/lib/votecounting.ts:412]: Would be nice to have a really basic test, at least.

**Review by eventualbuddha — APPROVED** [review]

**Pull request merged**